src/encoded/audit/experiment.py

```diff
@@ -1137,9 +1137,9 @@ def check_experiment_chip_seq_standards(
     if 'replication_type' not in experiment or experiment['replication_type'] == 'unreplicated':
         return
 
-    idr_metrics = get_metrics(idr_peaks_files, 'IDRQualityMetric') 
+    idr_metrics = get_metrics(idr_peaks_files, 'IDRQualityMetric')
```

Comment:
I think correcting trailing spaces is great, but please next time do it in a separate ticket.

Reply:
Let me revert this. I set auto trim in my editor and that's why.